internal/cmd/alpha/module/unmanage.go

```diff
@@ -46,5 +47,8 @@ func runUnmanage(cfg *unmanageConfig) clierror.Error {
 	if err != nil {
 		return clierror.Wrap(err, clierror.New("failed to check module state"))
 	}
+
+	fmt.Printf("Module %s set to unmanaged\n\n", cfg.module)
```

Comment:
I would suggest using a single new line char to make output consistent with other commands
```suggestion
	fmt.Printf("Module %s set to unmanaged\n", cfg.module)
```